include/sparkle_script.h

```diff
@@ -0,0 +1,20 @@
+#include "common.h"
+
+typedef s32 SparkleScript[0];
```

Comment:
Wondering if this should be unsized ie `s32 SparkleScript[];`, I recall issues with 0-sized EvtScripts.. does this match the other *Script types?

Reply:
This is the same as EvtScript, HudScript, and ItemScript.